jdi-light-angular-tests/src/main/java/io/github/com/entities/Users.java

```diff
@@ -0,0 +1,13 @@
+package io.github.com.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class Users extends DataClass<Users> {
+    public String firstName,
```

Comment:
Please change to 
    public String firstName;
    public String lastName;
    public String address;
    public String address2;
    public String city;
    public String state;
    public String postalCode;

Your variant it's not common for Java code https://rules.sonarsource.com/java/tag/convention/RSPEC-1659